openzaak/src/main/kotlin/com/ritense/openzaak/domain/mapping/impl/ZaakInstanceLinks.kt

```diff
@@ -16,7 +16,9 @@
 
 package com.ritense.openzaak.domain.mapping.impl
 
+import com.ritense.zakenapi.domain.ZaakInstanceLink
+
 class ZaakInstanceLinks : HashSet<ZaakInstanceLink> {
```

Comment:
is this really necessary?

Reply:
yes, 
```
import com.ritense.zakenapi.domain.ZaakInstanceLink
```
is needed because it is moved to a different module

Reply:
i mean do we need an object that is just an HashSet of ZaakInstanceLink

Reply:
Just checked it. This class is really old and isn't used anywhere. So i removed it